sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/OperationType.cs

```diff
@@ -1,17 +1,21 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
+
 namespace Azure.Monitor.OpenTelemetry.Exporter.Internals
 {
+    [Flags]
     internal enum OperationType
     {
-        Unknown,
-        Azure,
-        Common,
-        Db,
-        FaaS,
-        Http,
-        Messaging,
-        Rpc
-    };
+        Unknown = 0,
+        Azure = 1,
+        Common = 2,
+        Db = 4,
+        FaaS = 8,
+        Http = 16,
+        Messaging = 32,
+        Rpc = 64,
+        V2 = 128
```

Comment:
Have you considered adding a separate enum for schema version? Adding schema version to OperationType is looking odd.

Reply:
We are not planning to support multiple schema versions at this point. This is an interim solution to support both schema versions and remove the old one little later. 

If we want to consider adding schema version support, I recommend after GA. It's a big change and not just limited to an enum.

Reply:
Not necessarily an enum. But just a separate field on ActivityTagsProcessor which can be used to determine which transformation should we apply. That won't be a big change right? That would just change this check https://github.com/Azure/azure-sdk-for-net/pull/37357/files#diff-649090d26c1bac6ffac898f9a654d40e96d4409f2c7d85da680c7ffbda4457ff:~:text=Tags%5BContextTagKeys.AiOperationName.ToString()%5D%20%3D%20activityTagsProcessor.activityType.HasFlag(OperationType.V2)

This list for OperationType might grow when we have new conventions for other types

Reply:
nit: If we're making changes to this, consider using `v1_21_0` instead of `v2` to specify the exact semantic version we're referencing.

Reply:
As we discussed, will leave a TODO to decide if this change is needed in a follow up PR.